src/index.html

```diff
@@ -16,16 +17,19 @@
       href="style.css"
     />
   </head>
+
   <body>
     <input
+      class="search-bars search-bars-big"
       type="text"
-      data-qa="keypress"
+      data-qa="big"
       placeholder="Try “Los Angeles“"
     />
```

Comment:
The `input` element here has no associated `label` element. For accessibility purposes, it's recommended to use a `label` with a `for` attribute that matches the `id` of the `input`. It would make it more accessible for people using screen readers and also improves the user experience for those who can see the screen because clicking on the label will focus the `input`.